.github/workflows/integration-tests.yml

```diff
@@ -74,7 +74,7 @@ jobs:
             -   uses: ./.github/actions/setup
 
             -   name: Run Modulith CI Tests
-                timeout-minutes: 6
```

Comment:
I suppose these timeouts could go back to 4 minutes (in a normal startup 2 minutes is normally enough to get a success) at least for the runStartUpCheck when it runs as a separate step

Reply:
reverted back